### PR TITLE
feat: Navigation and 3-pane UI

### PR DESCRIPTION
**Pull request by louije**
Created a new layout: `app/views/layouts/viewer.html.erb` that serves as the skeleton for the "app" part of the app.
It contains multiple `<turbo-frame>` tags, nested inside `content_for` blocks.
This way, individual actions (such as `ContactsController#show` or `ConversationsController#index` or `TeamsController#edit`) can all work cleanly, whether from a fresh load or through a Turbo Frame navigation.

Made it so the main tab bar is only used on mobile too.

No more confusions on the main frame names:
- `navigation` is the left pane. Used for the menu and the conversations list.
- `primary` is the central pane. Used for most anything.
- `secondary` is the right pane. Used by inbox and presets.

Future controllers that rely on the Viewer (ie, the three-pane interface) should:

- Add `layout "viewer"` as a class level invocation in the controller.
- Put the content of their views (so, `index.html.erb`, `show.html.erb` and so on) inside a `<% content_for(:primary) do %>` or `<% content_for(:secondary) do %>` block, without worrying about the turbo frames.

The `viewer.html.erb` file is currently set up to autoload the conversations list if necessary. This is set in the `should_load_conversations?` method from `application_helper.rb`. So, no need to resort to shenanigans like "rendering index" the way I used to do for `ConversationsController#show`.

### CODE REVIEW

File: config/locales/fr.yml [hunk 1]
```diff
@@ -20,6 +20,15 @@ fr:
     partner_footer:
       logo-anct-alt: "Logo ANCT Société numérique"
       logo-vv-alt: "Logo Vives voies"
+    tab_bar:
+      home: "Accueil"
+      conversations: "Messagerie"
+      contacts: "Répertoire"
+      presets: "Modèles"
+      inbox: "Chat"
```
Comment (alallema) [inline config/locales/fr.yml:28]: Faudrait supprimer celle dans `conversations` du coup si on s'en sert plus

File: app/controllers/contacts_controller.rb [hunk 2]
```diff
@@ -29,18 +31,15 @@ def edit
...
     @team = @contact.team
 
     if @contact.save
-      respond_to do |format|
-        format.html { redirect_to team_contact_path(@team, @contact) }
-        format.turbo_stream {
-          flash.now[:notice] = I18n.t(".contacts.create.success")
-        }
-      end
+      redirect_to team_contact_path(@team, @contact), notice: I18n.t(".contacts.create.success")
```
Comment (alallema) [inline app/controllers/contacts_controller.rb:42]: Trop bien !

File: app/controllers/contacts_controller.rb [hunk 3]
```diff
@@ -17,7 +19,7 @@ def show
   def new
     @create_conversation = ActiveModel::Type::Boolean.new.cast(params[:create_conversation])
     @contact = Contact.new(team_id: @team.id)
-    if params[:modal]
+    if current_frame == "modal"
```
Comment (alallema) [inline app/controllers/contacts_controller.rb:22]: On est d'accord que ça ne servira que dans le cas de la création d'une nouvelle conversation sans contact ? On a pas d'UI pour ça si ?

**Review by alallema — APPROVED** [review]
Ca me semble super et ça marche très bien ✨

File: app/controllers/contacts_controller.rb (hunk 3 above)
Comment (louije) [reply app/controllers/contacts_controller.rb:22]: Absolument, c'est pour faire tourner l'UI actuelle qui doit être remplacée

**Comment by louije** [comment]
hop je merge. merci @alallema. Je trouve que `/conversations/index.html.erb` et `/conversations/show.html.erb` sont de bons exemples si tu cherches pour templates.

**Pull request merged**